lib/sycamore/sycamore/query/planner.py

```diff
@@ -76,6 +76,59 @@
 )
 
 
+def process_json_plan(
```

Comment:
What's the advantage of lifting this out of the Planner class?

Reply:
I'm using it as a util in the tests. Easier to create a plan and just have this convert it into the class